ci/test/00_setup_env_mac_cross.sh

```diff
@@ -17,4 +17,8 @@ export XCODE_BUILD_ID=15A240d
 export RUN_UNIT_TESTS=false
 export RUN_FUNCTIONAL_TESTS=false
 export GOAL="deploy"
-export BITCOIN_CONFIG="-DBUILD_GUI=ON -DREDUCE_EXPORTS=ON"
+export BITCOIN_CONFIG="\
+-DWITH_MULTIPROCESS=OFF \
```

Comment:
> TODO:
> 
>    * [ ]  fix `unknown target CPU 'armv8-a+crc+crypto'` [failure](https://cirrus-ci.com/task/5887765378760704) and re-enable macOS cross

Similar to https://github.com/bitcoin/bitcoin/pull/30975#issuecomment-2483394112, I do think that `-DWITH_MULTIPROCESS=OFF` can be dropped now as depends are [fixed](https://github.com/bitcoin/bitcoin/pull/30975#discussion_r1848171448).

Reply:
Good catch, pushed...